### DISCUSSION TITLE
Does PAG have multi-language support ?

### DISCUSSION THREAD

**Discussion by Meet-Miyani** [Tencent/libpag#487] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/91192898/185740744-53269cd7-6cb3-420a-b3a4-cbc06b30e9f7.png)

Does PAGViewer supports multiple languages ?
It's After Effects has the support of changing language ?
Actually I don't understand Chinese which make a bit difficult for me to understand, what the errors are trying to say, when exporting an After Effects file to PAG File. I found a folder in the PAG Viewer directory which says Translations and seems like there are some files which maybe provided for multi language support.
I tried looking through the PagViewer App and even the PAG Extension for the After Effects. But I didn't fount any option to make the Language changes.

Please Help

**Comment by domchen** [comment, marked as answer]
Hi @Meet-Miyani , Sorry about the inconvenience. We are currently working on adding multi-language support for PAG.  It will be released in a few weeks.

**Reply by Meet-Miyani** [reply]
Thank you much, for the support.
It will help a lot as I have spend pretty much time converting the errors into English for understanding. Hehe!

**Reply by Meet-Miyani** [reply]
> Hi@Meet-Miyani , Sorry about the inconvenience. We are currently working on adding multi-language support for PAG. It will be released in a few weeks.

Hello, any updates for the language support ?

**Reply by domchen** [reply]
The multi-language support have been released for a long time. Please upgrade the PAGViewer to the lastest version, and goto the perefences windows, make sure the [ use english] option is checked. Then restart the PAGViewer, you will good to go.

**Reply by Meet-Miyani** [reply]
![image](https://github.com/Tencent/libpag/assets/91192898/c4b5daa2-ee5a-4ec8-9349-ebedb8b0ce5a)

Hello, I am unable to update the PAGViewer. Not from the player itself neither from the website.
From website it keeps loading but download doesn't start.

**Reply by domchen** [reply]
It seems like there is something wrong with your network. Please check whether you are downloading over an VPN or other proxies, If so, turn them off. Or may you can switch to another network and try again.

**Reply by Meet-Miyani** [reply]
Yes, might the servers were not allowed in my country. I did it with a VPN later. Thank you
